atuin/src/command/server.rs

```diff
@@ -19,6 +19,9 @@ pub enum Cmd {
         #[clap(long, short)]
         port: Option<u16>,
     },
+
+    /// Print server example configuration
+    Config,
```

Comment:
maybe let's adjust this so the usage is `default-config`, that way it's clear that it's going to print the default config. Otherwise, the user may think they are able to get/set current config values via the cli